### ISSUE TITLE
Nested controller PR isn't complete

### ISSUE THREAD

**Issue by whitlockjc** [apigee-127/swagger-tools#427]
PR #422 added nested controller support but it's not quite complete.  It seems that the built-in controller name is wrong.  /cc @bitcloud

**Comment by ashtarcommunications** [comment]
Any update on this?

Would it at least be possible to get a release post the PR from #422? At the moment other packages like swagger-node are pegged to 0.10.1, so nested controllers don't work at all without some manual dependency management.

**Comment by whitlockjc** [comment]
Well, #422 was finished in #434 from what I can tell so all we need to do is get a release out?

**Comment by ashtarcommunications** [comment]
Yep, I think getting a release out would solve this issue. Other packages would still need to update their dependencies, but it would make it a lot easier...

**Comment by whitlockjc** [comment]
Okay.  I'll try to get one out ASAP.  Feel free to ping me if not done by EOD.

**Comment by ashtarcommunications** [comment]
Ping

**Comment by pekunicki** [comment]
ping @whitlockjc

**Comment by whitlockjc** [comment]
Sorry for the delay, i was hoping to get a `json-refs` release out first.
